### ISSUE TITLE
Feature: Replica Set and SSH

### ISSUE THREAD

**Issue by simsekgokhan** [Studio3T/robomongo#1340]
Implement SSH support for replica sets.

Duplicate:
https://github.com/Studio3T/robomongo/issues/1285

**Comment by pauln** [comment]
Hi @simsekgokhan, has there been any progress on this?  Robo 3T 1.1.1 still doesn't seem to support replica sets over SSH; is there at least a plan for when this will be implemented?

**Comment by jobinthomasnypl** [comment]
@pauln Robo3T 1.2.1 also has this issue.

**Comment by nickdoikov** [comment]
Guys, any idea whet this feature  will be implemented ?

**Comment by mohag** [comment]
Robo 3T 1.3.1 also still has this problem

**Comment by paramaggarwal** [comment]
It was mentioned that till this is implemented, we can use connecting to primary directly as a fallback - is anyone familiar with how to do this? Should I skip giving the `replicaSet` parameter?

**Comment by alitoufighi** [comment]
@paramaggarwal 
Yes. You have to connect to Primary instance in `Direct Connection` mode. If you don't know what the IP and port of your Primary is, simply connect to an instance in Direct Connection mode, then run `rs.isMaster()` command on it. It will return an object which has a `primary` field. It will show you where your Primary is:

![image](https://user-images.githubusercontent.com/30089812/92641625-44e4aa80-f2f4-11ea-8038-30bb1c69da12.png)
